Rob Wilton comments (editorial) (#509)
ietf-wg-jsonpath/draft-ietf-jsonpath-base: draft-ietf-jsonpath-base.md

### Examples

<aside markdown="1">
In this and the following examples in Sections {{<root-identifier}} and
{{<selector-details}} except for {{tbl-comparison}}, we will present a
JSON text to show the JSON value used as the query argument to the
queries in the examples, and then a table with the columns:

* Query: an example query to be applied to the query argument
* Result: the query result as a list of JSON values that were located in the query argument
* Result Path: the query result as a list of (normalized) paths into

>   the query argument, giving locations in parallel to the previous column

Reviewer: ```suggestion
  the query argument, giving locations of the JSON values in the previous column
```

Reviewer: Do we want a full stop at the end of the last bullet?

Author: > Do we want a full stop at the end of the last bullet?

Not normally.  We either use punctuation throughout the list or leave it off entirely.
There are no sentences here, so the latter is the usual choice.

The examples in {{tbl-name}} show the name selector in use by child segments:

| Query                | Result       | Result Paths            | Comment                      |
| :---:                | ------       | :----------:            | -------                      |

> | `$.o['j j']`        | `{"k.k": 3}` | `$['o']['j j']`        | Named value in nested object |

Reviewer: It's probably accurate, but having the same comment for this as for the following two rows doesn't read very well.

Author: Yes, I wondered about that.  "nested object" fits both a single and a dual level of nesting...

Author: See commit below...
